db/model/Variable.ts

```diff
@@ -491,10 +468,15 @@ const httpsAgent = new https.Agent({
 export const fetchS3Values = async (
     variableId: OwidVariableId
 ): Promise<OwidVariableMixedData> => {
-    const dataPath = await getOwidVariableDataPath(variableId)
+    const { dataPath, metadataPath } = await getOwidVariableDataAndMetadataPath(
+        variableId
+    )
     if (!dataPath) {
         throw new Error(`Missing dataPath for variable ${variableId}`)
     }
+    if (!metadataPath) {
+        throw new Error(`Missing metadataPath for variable ${variableId}`)
+    }
```

Comment:
Should that be an error right now, when we're not relying on `metadataPath` yet?

Reply:
If data JSON exists then metadata JSON should exist too. So this is more of an assert to make sure I haven't missed something and is a preparation for future.